crates/sui-pg-db/src/lib.rs

```diff
@@ -195,10 +222,13 @@ async fn pool(database_url: Url, args: DbArgs) -> anyhow::Result<Pool<AsyncPgCon
 #[cfg(test)]
 mod tests {
     use super::*;
+
     use diesel::prelude::QueryableByName;
     use diesel_async::RunQueryDsl;
     use diesel_migrations::EmbeddedMigrations;
 
+    use crate::temp;
```

Comment:
Surprised that you need this, because the `temp` should be part of the `super::*` import.